sorc/ncep_post.fd/CTLBLK.f

```diff
@@ -54,7 +54,7 @@ module CTLBLK_mod
           SPL(komax),ALSL(komax),PREC_ACC_DT,PT_TBL,PREC_ACC_DT1,spval
 ! real :: SPVAL=9.9e10                                     ! Moorthi
 !
-  integer :: NUM_PROCS,ME,JSTA,JEND,JSTA_M,JEND_M,                     &
+  integer :: NUM_PROCS,ME,JSTA,JEND,ista,iend,JSTA_M,JEND_M,                     &
              JSTA_M2,JEND_M2,IUP,IDN,ICNT(0:1023),IDSP(0:1023),        &
              JSTA_2L, JEND_2U,JVEND_2u,NUM_SERVERS, MPI_COMM_INTER,    &
```

Comment:
@GeorgeVandenberghe-NOAA Will you consider adding ISTA_2L and IDEN_2u for x decomposition?